docs/how_to/layer_manipulator.md

```diff
@@ -8,23 +8,29 @@ Sometimes objects in your data have a systematic shift relative to one another.
 
 ## Instructions
 
-1. Open the [layer_manipulator.py](https://napari-threedee.github.io/generated/gallery/plugin/layer_manipulator_plugin/) example from the docs. The viewer is initalized in 3D rendering mode. You can 
-   click and drag in the canvas to rotate the view. Note that the two layers are the same image, but not aligned.
+1. To run this example, first download the script from the [Examples Gallery page](https://napari-threedee.github.io/generated/gallery/plugin/layer_manipulator_plugin/); the link is at the bottom. Ensure the location is somewhere memorable. Or, if you've cloned this repository, the script can be found at `napari-threedee/docs/examples/plugin/layer_manipulator_plugin.py`. 
+
+2. Ensure you have the appropriate virtual environment active, change directories to the script location, and then you can execute the script using:
```

Comment:
Same comment as in previous PRs: I think "the appropriate virtual environment" is not an easy instruction with no further details provided for beginners.

I don't think this should block the docs PRs you have going on, but do you want to make an issue about this, so it can be improved down the track?

Reply:
I think this is a good point, @GenevieveBuckley . I think a future improvement would be to add an "installation" section that discusses the usage of virtual environment. We can then link to that page from the "activate environment" instruction.

Reply:
Right now the n3d installation docs link to the napari docs, which talk about virtual environments. Those docs have been updated to also link to other resources about envs, like Talley's guide.
In case you've not seen this, here's what will go live with 0.5.0:
https://napari.org/dev/tutorials/fundamentals/installation.html#install-as-python-package-recommended
And to make life easier, here's the info box about envs:
![image](https://github.com/napari-threedee/napari-threedee/assets/76622105/a2aeaa63-95f8-491e-ba1b-39ef16bccfe4)

Reply:
If you want this could be duplicated into these docs, but this is a napari plugin, so I think deferring to the napari docs is fine?